frontend/packages/shared/src/components/altinnHeaderButtons/AltinnHeaderButton.module.css

```
@@ -1,27 +1,51 @@
-.properties {
+.linkButton {
+  display: flex;
+  justify-content: center;
   align-items: center;
-  padding-right: 8rem;
-  gap: 1rem;
+  border: var(--fds-border_width-default) solid transparent;
+  padding: 0 var(--fds-spacing-3);
+  cursor: pointer;
+  font-family: inherit;
+  text-decoration: none;
+  position: relative;
+  border-radius: var(--fds-border_radius-interactive);
+  min-height: var(--fds-sizing-8);
 }
 
-.rightContentButtons {
-  align-items: center;
-  display: flex;
-  gap: 1rem;
+.linkButton:focus-visible {
+  --fds-focus-border-width: 3px;
+
+  outline: var(--fds-focus-border-width) solid var(--fds-semantic-border-focus-outline);
+  outline-offset: var(--fds-focus-border-width);
+  box-shadow: 0 0 0 var(--fds-focus-border-width) var(--fds-semantic-border-focus-boxshadow);
+}
+
+.invertedButton {
```

I suggest overriding the `visited` pseudo-class to remove the purple color as mentioned in my previous comment:

```suggestion
.invertedButton,
.invertedButton:focus,
.invertedButton:visited {
```

I also suggest overriding the `focus` pseudo-class to remove the yellow color when focusing the link:

https://github.com/Altinn/altinn-studio/assets/24462611/831e3014-457d-49a6-bd19-da6da8e260c4

---

I totally agree, thank you!